pinot-common/src/main/java/com/linkedin/pinot/common/data/FieldSpec.java

```diff
@@ -48,6 +54,7 @@
   private static final Float DEFAULT_METRIC_NULL_VALUE_OF_FLOAT = 0.0F;
   private static final Double DEFAULT_METRIC_NULL_VALUE_OF_DOUBLE = 0.0D;
   private static final String DEFAULT_METRIC_NULL_VALUE_OF_STRING = "null";
+  private static final Bytes DEFAULT_DIMENSION_NULL_VALUE_OF_BYTES = new Bytes(new byte[0]);
```

Comment:
Please add both DEFAULT_DIMENSION_NULL_VALUE_OF_BYTES and DEFAULT_METRIC_NULL_VALUE_OF_BYTES

Reply:
Ok, will do.